apps/explorer/src/api/assets/assetsHooks.ts

```diff
@@ -172,9 +173,7 @@ export const useAssetsMetadata = (assetIds: string[] = []) => {
   } else return returnedCompleteMetadata
 }
 
-// We should get this list from the backend
-// See: https://github.com/alephium/explorer-backend/issues/512
-export const useTokensWithAvailablePrice = () => ['ALPH', 'USDT', 'USDC', 'DAI', 'WBTC', 'WETH', 'AYIN']
+export const useTokensWithAvailablePrice = () => Object.values(explorer.TokensWithPrice) as string[]
```

Comment:
I don't see why this should be a hook.

On desktop wallet we do this:

```
const pricedTokens = Object.keys(explorer.TokensWithPrice)
```

Maybe we can simply export this from `@alephium/shared` instead?

Reply:
Yep, let's do that.